archey/entries/cpu.py

```diff
@@ -4,13 +4,17 @@
 
 from subprocess import check_output
 
+from archey.entry import Entry
 
-class CPU:
+
+class CPU(Entry):
     """
     Parse `/proc/cpuinfo` file to retrieve the model name.
     If no information could be retrieved, calls `lscpu`.
     """
-    def __init__(self):
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
```

Comment:
For each one of our current entries, instantiation parameters will now be passed to the base class. Tell me what you think about that too ; I guess/hope it will help us in the future.

Reply:
This seems good, it could potentially allow us to add more than `name` and `value` attributes to entries quite easily without having to modify all files to pass those parameters to our `Entry` abstract class. I can't think of anything right now that might need this but, it's better than just hardcoding `name` and `value` anyway 👍